README.markdown

```diff
@@ -280,7 +280,15 @@ $redis->swapdb(0, 1); /* Swaps DB 0 with DB 1 atomically */
 
 ### close
 -----
-_**Description**_: Disconnects from the Redis instance, except when `pconnect` is used.
+_**Description**_: Disconnects from the Redis instance, include when `pconnect` is used.
+
+*Note*: Requires Redis >= 4.2.0, phpredis can closed the persistent connection.
```

Comment:
I'm not an expert bu may be `can close` instead `can closed`?

Reply:
updated